src/utilcode/sstring.cpp

```diff
@@ -86,11 +74,9 @@ static WCHAR MapChar(WCHAR wc, DWORD dwFlags, LocaleID lcid)
 #ifndef FEATURE_PAL
     
 #ifdef FEATURE_USE_LCID
```

Comment:
Aren't you removing `FEATURE_USE_LCID`?

Reply:
I have removed `FEATURE_USE_LCID` from managed side only. There was mismatch between how it was defined on managed side and unmanaged side: The managed side had it defined (on Windows), but the unmanaged side has it undefined.

Most of the remaining uses of `FEATURE_USE_LCID` on the unmanaged side are around magic LCID argument marshaling for COM. The magic LCID argument marshaling does not work today because of `FEATURE_USE_LCID` is not defined. Do you happen to know whether it is something we should fix?

Reply:
Opened https://github.com/dotnet/coreclr/issues/24629

Reply:
Sorry @jkotas. Apparently I didn't hit 'Comment' last night. The LCID in COM isn't something I have used much at all and personally have avoided because localization in via Win32 APIs is a pain. The filed issue is appropriate in this case, but I will have to educate myself on the details of LCID usage in COM before commenting on enabling it - my guess is Office is a consumer of the feature.